datahub-web-react/src/app/analytics/analytics.ts

```diff
@@ -7,15 +7,25 @@ import { getBrowserId } from '../browserId';
 
 const appName = 'datahub-react';
 
+export const THIRD_PARTY_LOGGING_KEY = 'isMixpanelEnabled';
```

Comment:
should this say `isThirdPartyLoggingEnabled`?

Reply:
i'm a dummy haha